.github/workflows/test.yaml

```diff
@@ -50,8 +54,8 @@ jobs:
       - name: Check out Java SDK
         uses: actions/checkout@v4
         with:
-          repository: Eppo-exp/android-sdk
-          ref: ${{ env.SDK_BRANCH}}
+          repository: ${{ inputs.base_repo || github.repository }}
```

Comment:
When we re-use this workflow in the sdk-test-data repository, `github.repository` refers to the `sdk-test-data` repository instead of android-sdk, so we explicitly defined which repo to test. This prevents the test from running on a fork of the repo